deposit/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -110,6 +110,21 @@
         </constructor-arg>
         <constructor-arg name="numWorkers" value="${workers.high.num:4}" />
     </bean>
+
+    <bean id="migrationDepositWorkPool" class="net.greghaines.jesque.worker.WorkerPool">
+        <constructor-arg name="workerFactory">
+            <bean class="net.greghaines.jesque.worker.WorkerImplFactory">
+                <constructor-arg name="config" ref="jesqueConfig" />
+                <constructor-arg name="queues">
+                    <util:list>
+                        <value>PREPARE_MIGRATION</value>
+                    </util:list>
+                </constructor-arg>
+                <constructor-arg ref="jobFactory" />
+            </bean>
+        </constructor-arg>
+        <constructor-arg name="numWorkers" value="${workers.high.num:4}" />
```

Comment:
Could you give this a separate property, probably `workers.migration.num` and set its default to 2?